sandbox/public/index.html

```diff
@@ -14,8 +14,8 @@
     <title>Mock website hosting the Alloy</title>
 
     <script>
-      (function(a,b){b.forEach(function(c){if(!a[c]){a.__adobeNS=a.__adobeNS||[],
-      a.__adobeNS.push(c),a[c]=function(){a[c].q.push(arguments)},a[c].q=[]}})})
+      (function(a,b){b.forEach(function(c){if(!a[c]){a.__alloyNS=a.__alloyNS||[],
+      a.__alloyNS.push(c),a[c]=function(){a[c].q.push(arguments)},a[c].q=[]}})})
       (window,['alloy','ebda'])
```

Comment:
Since most customers will have a single instance, should we support passing the namespace as a string, and only use an array if you need multiple?

```
...(window,'alloy')...

&
...(window,['alloy','ebda'])...
```

Reply:
The more I think about this, it doesn't seem necessary to change the array. Actually, defaulting to an array might be a good thing, because it'll educate the user that this library supports multiple instances.

Reply:
I went through the same thought process. I actually built it that way initially (string unless you have multiple), but in the end thought it might be more straightforward for a user to just tack on another string rather than understand they need to change it to an array.